app/components/edit/edit_icon_button_component.rb

```diff
@@ -3,16 +3,18 @@
 module Edit
   # Component for a pencil icon button to edit a draft
   class EditIconButtonComponent < Elements::IconButtonLinkComponent
-    def initialize(presenter:, icon_classes:, **args)
+    def initialize(presenter:, icon_classes: [], tab: nil, **args)
```

Comment:
Just curious: I only see one use of `EditIconButtonComponent` in H3, and it does not inject `icon_classes`. Why we do we need that arg here?  Maybe I've missed some metaprogramming?

Reply:
It's cruft. Removing.